acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/pages/site/HomePage.java

```diff
@@ -15,7 +15,13 @@ public HomePage(WebDriverProvider webDriverProvider, final PageHelper helper) {
     }
 
     public String getPageTitle(){
-        return getWebDriver().findElement(By.xpath("html/body/header/div/div/section/div/form/div[2]/button")).getText();
+        try {
+            Thread.sleep(2000);
```

Comment:
Can you get rid of the sleeps? Assume this was just for testing?

Reply:
Done. Thanks Jacob! :)